src/logic/calculate.js

```diff
@@ -0,0 +1,18 @@
+import operate from './operate';
+
+const calculate = ({
```

Comment:
The calculate function should receive two arguments, data and buttonName or buttonVal in your case; data should be an object containing three properties: next, total and operation, as you can see from the [Pathwright to this Milestone](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896094/), this means taking two arguments, and not one object as argument with four properties.
![image](https://user-images.githubusercontent.com/46115725/73101877-44f37f80-3eb6-11ea-94fe-3057e96b30df.png)